shared/chat/conversation/messages/index.stories.js

```diff
@@ -4,10 +4,10 @@ import placeholder from './placeholder/index.stories'
 import reactButton from './react-button/index.stories'
 import reactionTooltip from './reaction-tooltip/index.stories'
 import text from './text/index.stories'
-import walletPayment from './wallet-payment/index.stories'
+import accountPayment from './account-payment/index.stories'
 
 const load = () => {
-  ;[chooseEmoji, placeholder, reactButton, reactionTooltip, text, walletPayment].forEach(load => load())
+  ;[chooseEmoji, placeholder, reactButton, reactionTooltip, text, accountPayment].forEach(load => load())
```

Comment:
It may be worth changing the name of the story in `account-payment/index.stories.js` from `Chat/Conversation/Wallet payments` to `Chat/Conversation/Account payments` too